starter-s3.yaml

```diff
@@ -118,16 +118,26 @@ data:
 
     tide:
       queries:
-      - labels:
+      - orgs:
+        - kitex-contrib
+        excludedRepos:
```

Comment:
em. new repos won't be excluded, right?  We can give it a try.

Reply:
yes, I just wanna have a try.